pages/managed-cards/index.vue

```diff
@@ -88,8 +80,12 @@ export default class CardsPage extends mixins(BaseMixin, CardsMixin, KyVerified)
     return undefined
   }
 
+  get cardStateFilters() {
```

Comment:
I think this getter name is a bit misleading. Can you rename it in a way that it reflects that the cards with the appropriate states are retrieved? Something along the lines of includeDestroyedCards()